pkg/server/server.go

```diff
@@ -60,14 +60,17 @@ func NewServer(config *viper.Viper) *Server {
 		environment: environment,
 	}
 
-	// set up routes
-	s.routes(
+	// set up middleware
+	s.router.Use(
 		newContextMiddleware(),
 		newCORSMiddleware(clientAddress),
 		newTraceMiddleware(),
 		newLoggerMiddleware(s.logger, environment),
 	)
```

Comment:
OK I see now, we've moved the middleware higher up in the chain in a sense instead of having them buried in the routes file.